internals/schemas/app.schema.json

```diff
@@ -138,6 +138,10 @@
         "enableForwardIncidentToExternal": {
           "description": "Enables forwarding the handling of an incident to an external organization.",
           "type": "boolean"
+        },
+        "showRealTimeDashboard": {
+          "description": "Shows the realtimedashboard",
```

Comment:
should be realtime dashboard I guesss ( camelCase van be read like camel case)

Reply:
dito